fixtures/demo.json

```diff
@@ -1100,6 +1100,7 @@
       "collection": 1,
       "title": "frank",
       "file": "original_images/frank.png",
```

Comment:
I don't know where the image is from 🤔

Reply:
Me either. We might want to replace it with an image from the bakery demo to be safe.